oviewer/input_search_test.go

```diff
@@ -278,14 +278,14 @@ func Test_stripBackSlash(t *testing.T) {
 		want string
 	}{
 		{
-			name: "nonEsacpe",
+			name: "nonEsacape",
```

Comment:
This no?

```suggestion
			name: "nonEscape",
```

Or is it intentional?

Reply:
Oops